Mathlib/Algebra/GroupWithZero/Units/Basic.lean

```diff
@@ -94,6 +94,10 @@ theorem inverse_unit (u : M₀ˣ) : inverse (u : M₀) = (u⁻¹ : M₀ˣ) := by
   rw [inverse, dif_pos u.isUnit, IsUnit.unit_of_val_units]
 #align ring.inverse_unit Ring.inverse_unit
 
+theorem inverse_isUnit (x : M₀) (h : IsUnit x) : inverse x = ((h.unit⁻¹ : M₀ˣ) : M₀) := dif_pos h
```

Comment:
```suggestion
theorem inverse_of_isUnit (x : M₀) (h : IsUnit x) : inverse x = ((h.unit⁻¹ : M₀ˣ) : M₀) := dif_pos h
```
though actually now that I see this lemma I'm less sure that it's useful

Reply:
we do need this lemma, it's one of the ones used in the proof about mapping ring.inverse

Reply:
BTW `x` can be implicit.